packages/office-ui-fabric-react/src/utilities/dateMath/DateMath.ts

```diff
@@ -207,11 +205,78 @@ function getDatePartHashValue(date: Date) {
  * @param {DayOfWeek} firstDayOfWeek - The first day of the week (0-6, Sunday = 0)
  * @return {Number} The week's number in the year.
  */
-export function getWeekNumber(date: Date, firstDayOfWeek: DayOfWeek) {
-  let num = getDayOfYear(date) - 1;
-  let num2 = date.getDay() - (num % 7);
-  let num3 = ((num2 - firstDayOfWeek) + 14) % 7;
-  return Math.floor(((num + num3) / 7) + 1);
+export function getWeekNumber(date: Date, firstDayOfWeek: DayOfWeek, firstWeekOfYear: FirstWeekOfYear) {
```

Comment:
Kelly had already added code for week numbers. Are you adding your own also? We should have one way to do week numbers. Kelly was using the method I recommended based on JSMVVM implementation so I suggest we stick with that and just augment with the firstWeekOfYear setting support

Reply:
That's exactly what i'm doing. Her function is getDefaultWeekNumber() - I renames variables to make a little more sense of the math.

Reply:
Also, we should add unit tests